app/views/admin/person_uploads/_form.html.haml

```diff
@@ -0,0 +1,21 @@
+= form_for [:admin, @upload], multipart: true do |f|
+
+  .form-group{ :class => ('gov-uk-field-error' if f.object.errors.include?(:file)) }
```

Comment:
I feel bad about this excessive duplication, which also exists in other forms. I think we should fix it everywhere as a separate piece of work.